adm_program/languages/de.xml

```diff
@@ -125,6 +125,7 @@
     <string name="DAT_VIEW_MODE_DETAIL">Detailliert</string>
     <string name="DAT_VIEW_MODE_COMPACT">Kompakt</string>
     <string name="DAT_VISIBILITY">Sichtbarkeit</string>
+    <string name="DAT_PHOTO_RIGHTS">Bei Auswahl eines Albums können nur noch Teilnehmer dieses Events an dieses Photoalbum</string>
```

Comment:
This sentence sounds strange.

Reply:
rephrased and english text added to en.xml